### ISSUE TITLE
no-nill false positive

### ISSUE THREAD

**Issue by graingert** [jfmengels/eslint-plugin-fp#28]
```js
 switch (ham) {
    case "spam": return 3;
    case blockTypes.skew:
    default: return 4;
  }
```

should pass
